src/common/schema-validator.ts

```diff
@@ -3,7 +3,7 @@ import * as assert from 'assert'
 const {Validator} = require('jsonschema')
 import {ValidationError} from './errors'
 import {isValidClassicAddress, isValidXAddress} from 'ripple-address-codec'
-import {isValidSecret} from './utils'
+import {isValidSecret} from '../utils/index'
```

Comment:
This should just be `../utils` - the `index` part isn't necessary, all index files are just the "home" for the folder

Reply:
Removed extra /index from all utils imports